src/WebJobs.Script/Config/FunctionsHostingConfigSource.cs

```diff
@@ -0,0 +1,42 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.Configuration.Json;
+
+namespace Microsoft.Azure.WebJobs.Script.Config
+{
+    /// <summary>
+    /// Represents a JSON file as an <see cref="IConfigurationSource"/>.
+    /// </summary>
+    public class FunctionsHostingConfigSource : FileConfigurationSource
+    {
+        private readonly string _path;
+
+        public FunctionsHostingConfigSource(IEnvironment environment)
+        {
+            _path = environment.GetEnvironmentVariable(EnvironmentSettingNames.FunctionsPlatformConfigFilePath);
```

Comment:
please add comments here on the possible values of the file path when running on Linux SKUs